src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -49,7 +51,16 @@ static const char* NoError = "None";
 // 0: success, 1: fail, 2: wait for async request
 int init()
 {
+    // Load default font
+    Text::load(fontName, reinterpret_cast<const char*>(default_font), default_font_size, "ttf", false);
+
 #ifdef THORVG_WG_RASTER_SUPPORT
+    // Check if WebGPU is supported in the browser
```

Comment:
This doesn't seem related to font support. Please separate this change into an additional commit.